### DISCUSSION TITLE
Green Software Practices catalog available ?

### DISCUSSION THREAD

**Discussion by DannyPROV** [Green-Software-Foundation/discussions#7] [Ask Anything] [answered]
Hi, there is a  'Green Software Patterns' catalog  on GSF portal but i can't find the 'Green Software Practices' catalog.
Is this not yet available ?
Thanks in advance for your help.
Danny

**Comment by jawache** [comment]
Hia @DannyPROV, the "practices" is available at learn.greensoftware.foundation and the patterns at patterns.greensoftware.foundation - do we have some links that are broken? Let me know :)

**Reply by DannyPROV** [reply]
Thank you Asim. My question is specifically to the CATALOG which is missing for the PRACTICES (A green software practice is a pattern applied to a specific vendor's product and informs practitioners about how to use that product in a more sustainable way.)The catalog exists for the PATTERNS but not for the PRACTICES ? See pyramid image on https://learn.greensoftware.foundation/introduction

**Reply by jawache** [reply, marked as answer]
Ahh sorry I got practices and principles mixed up. So the GSF won't be creating any practices, at least that's what the team agreed - the vendor specific content we believe should exist on vendor specific sites. Microsoft has done a good job of this with their Well Architected content, it links back to ours but is vendor specific to the microsoft practice. https://learn.microsoft.com/en-us/azure/well-architected/sustainability/

**Reply by DannyPROV** [reply]
Thank you Asim, that answers my question. The pyramid image for the PRACTICES , same as for the PATTERNS,  refers also to a catalog so maybe this needs updated to not cause confusion ?
